osmnx/geometries.py

```diff
@@ -353,8 +349,8 @@ def _create_gdf(response_jsons, polygon, tags):
 
     # else if there were elements in the response
     else:
-        # log creation
-        utils.log("Converting elements to geometries")
+        count = sum(len(rj["elements"]) for rj in response_jsons)
+        utils.log("Converting {count} elements in JSON responses to geometries")
```

Comment:
Making this a proper f-string will probably fix the tests:
```suggestion
        utils.log(f"Converting {count} elements in JSON responses to geometries")
```